app/src/main/java/com/kickstarter/libs/ExperimentsClientType.kt

```diff
@@ -57,7 +57,7 @@ interface ExperimentsClientType {
      * "session_variants_optimizely" : [ "Experiment1":"variant1",
```

Comment:
Update also the comments if the expected format has change to "Experiment[variant]" please :)